### DISCUSSION TITLE
How to retry failed Job?

### DISCUSSION THREAD

**Discussion by ghost** [OptimalBits/bull#2149] [Q&A] [answered]
## Scenario 
I have pushed almost 20k job in the queue. As I was in tight schedule I didn't add any failed job handling mechanism. So now I have 7k failed job in the queue. Which I can confirm by looking at Redis

## What I want. 
I want to push those job back to the queue again

## What I have tried so far
I saw few solution here and there which didn't fulfill my requirement

I will be glad if anyone can help me. Or can give me a guidance on how can I achieve that

**Comment by manast** [comment, marked as answer]
You could just use the getters to retrieve all failed jobs and then call "retry" on them.

**Reply by ghost** [reply]
That helps. Thank you
